File: 1-two-sum/1-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        new_dic = {}
        
        for i, n in enumerate(nums):
            result = target - n
            if result in new_dic:
                return [new_dic[result], i]
            new_dic[n] = i

File: 1-two-sum/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    
    let newDict = {}
    
    for (let i = 0; i < nums.length; i++){
        if (target - nums[i] in newDict) {
            return [newDict[target - nums[i]], i];
            
        }
        else{
            newDict[nums[i]] = i;
        }
    }
};